implementations/micrometer-registry-new-relic/src/main/java/io/micrometer/newrelic/NewRelicNamingConvention.java

```diff
@@ -45,16 +47,16 @@ public NewRelicNamingConvention(NamingConvention delegate) {
 
     @Override
     public String name(String name, Meter.Type type, @Nullable String baseUnit) {
-        return StringEscapeUtils.escapeJson(toValidNewRelicString(delegate.name(name, type, baseUnit)));
+        return toValidNewRelicString(delegate.name(name, type, baseUnit));
```

Comment:
With the addition of the APM integration, there was a potential issue where attributes would either be double JSON escaped, or never JSON escaped.
Removing the JSON escaping from the `NamingConvention` and only performing it when the data is actually converted into JSON prevents this from happening.